### PR TITLE
Ap 3001 threshold changes

### PR DESCRIPTION
**Pull request by willc-work**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-3001)

Add new threshold file
Update tests to cover either side of the new thresholds
Added new tests to the CFE integration tests

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by kmahern — APPROVED** [review]
lgtm 👍

File: spec/services/calculators/dependant_allowance_calculator_spec.rb [hunk 1]
```diff
@@ -141,50 +141,50 @@ module Calculators
       end
     end
 
-    # Changing these to be 2021 dates.
```
Comment (colinbruce) [inline spec/services/calculators/dependant_allowance_calculator_spec.rb:144]: It's hard to tell in the github view, but do we have a set of tests for each threshold file?  
I seem to remember this biting us when someone changed an old threshold file and the tests didn't fail.
I would suggest having a 2021 block, and add a 2022 block so we can test both thresholds

Comment (willc-work) [reply spec/services/calculators/dependant_allowance_calculator_spec.rb:144]: no tests for each threshold file, just to straddle the most recent threhsold period. I will add tests for all dates

Comment (colinbruce) [reply spec/services/calculators/dependant_allowance_calculator_spec.rb:144]: I wouldn't necessarily worry about _all_ dates, but I would definitely add new years. 
I seem to remember that someone did a search and replace on a value and changed multiple threshold files and we didn't spot it! 😬

Comment (willc-work) [reply spec/services/calculators/dependant_allowance_calculator_spec.rb:144]: fixed

**Pull request merged**